### PR TITLE
Changes to Pull Request Template

### PR DESCRIPTION
**Pull request by Anvilman6**
Hybridizes Rob's #11633 and changes put forward by @alao3alao3 on the Yog Discord, along with my own edits, per Jamie's request https://github.com/yogstation13/Yogstation/pull/11633#issuecomment-872080638

:cl:   
tweak: PR template changes
/:cl:

### CODE REVIEW

**Comment by adamsong** [comment]
You just had to break the maps didn't you?

**Comment by Anvilman6** [comment]
Wish I knew why changing the PR template caught all the maps on fire but such is coding

**Comment by adamsong** [comment]
`Octokit.NotFoundException: repositories/120966336/pulls/11683/files was not found.`
Its an error with the map diff program

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 
-### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
-For example, "This new chem will heal X brute damage". 
+Remove this text and write all information regarding your changes that should be known and documented through the Yogstation Wiki. 
+Important documentation information includes, but is not limited to: any numberical values that have been changed, any images that have to be updated, names of specific pages that will be impacted by your changes. 
 
 # Changelog
 
-Edit this changelog for changes that are noticeable by the players. Remove it if this isn't the case. If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name. Prefix the PR title with [admin] if it's something admin related. Prefix the PR title with [s] if you are fixing an exploit so it is not announced on discord and the server.
+Edit the changelog at the bottom for changes that are noticeable by the players. Remove it if this isn't the case.
```
Comment (patpol4) [inline .github/PULL_REQUEST_TEMPLATE.md]: We require a changelog for all changes just fyi

Comment (Anvilman6) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good point, I just copy pasted that from Rob's changes, so I'll modify it

**Pull request merged**